### PR TITLE
Add Crust Network telemetry icon

### PR DESCRIPTION
**Pull request by MyronFanQiu**
Follow the commit history, we add crust network's telemetry icon. `Crust Network CC2` is the name of our chain_spec.json. Please correct me if I miss something. Fix crustio/crust#336.

### CODE REVIEW

File: frontend/src/components/List/Column/ImplementationColumn.tsx [hunk 1]
```diff
@@ -46,6 +47,7 @@ const ICONS = {
   Totem: totemIcon,
   'Nodle Chain Node': nodleIcon,
   subzero: zeroIcon,
+  'Crust Network CC2': crustIcon,
```
Comment (maciejhirsz) [inline frontend/src/components/List/Column/ImplementationColumn.tsx:50]:
```suggestion
  Crust: crustIcon,
```
It should match the name of the node, not the chain:
Screenshot from 2020-12-10 09-36-36

**Review by maciejhirsz — CHANGES_REQUESTED** [review]

File: frontend/src/components/List/Column/ImplementationColumn.tsx (hunk 1 above)
Comment (MyronFanQiu) [reply frontend/src/components/List/Column/ImplementationColumn.tsx]: Thank you. I have fixed this.

**Review by maciejhirsz — APPROVED** [review]
LGTM

**Comment by maciejhirsz** [comment]
It's live now.

**Pull request merged**